ui/appui-react/src/appui-react/popup/ComponentPopup.tsx

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type { Orientation, SizeProps } from "@itwin/core-react";
+import { DivWithOutsideClick, Point, Size } from "@itwin/core-react";
+import * as React from "react";
+import { CursorPopup } from "../cursor/cursorpopup/CursorPopup";
+import type { PopupPropsBase } from "./PopupManager";
+import { PopupManager } from "./PopupManager";
+import { PositionPopup } from "./PositionPopup";
+import type { OnCancelFunc, RelativePosition } from "@itwin/appui-abstract";
+import { useEffect, useState } from "react";
+import { WrapperContext } from "../configurableui/ConfigurableUiContent";
+
+/**
+ *
+ */
+export interface ComponentPopupProps extends Omit<PopupPropsBase, "el"> {
```

Comment:
Let's not export Props types, since `React.ComponentProps<typeof ComponentPopup>` type helper is available and it makes it easier to stay backwards compatible.

Reply:
Fixed.